### PR TITLE
Productpage

### PR DESCRIPTION
**Pull request by JaanviNL**
Screenshot (27)

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -5,12 +5,34 @@
...
     display: flex;
     padding: 1rem 1rem 0.5rem 2rem;
     justify-content: space-between;
     align-items: center;
+    width: 81rem;
+    height: 5rem;
+    position: sticky;
+    top:0;
+    left:0;
+    z-index:999;
```
Comment (Rohitprasad83) [inline index.css:34]: why such hight z-index value?, z-index of 2 or 3 would be more than enough for navbar

File: store.css [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+.flex-column {
+    display: flex;
+    flex-direction: column;
+    gap: 0.3rem;
+    margin-top: 1rem;
+
+}
+.break-line
+{margin-top: 1rem;
```
Comment (Rohitprasad83) [inline store.css:41]: use a code formatter like prettier or beautify for consistent code format.

File: store.css [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+.product-card {
+    width: 14rem;
+    border: 1px solid var(--mediumgrey);
+    background-color: var(--grey);
+    padding: 0.6rem;
+    box-shadow: 3px 5px 5px var(--mediumgrey);
+    display: flex;
+    flex-direction: column;
+    cursor: pointer;
+
```
Comment (Rohitprasad83) [inline store.css:130]: remove extra whitespace

**Review by Rohitprasad83 — APPROVED** [review]
Looks good, just need few improvements.

**Review by aurobindoGupta — APPROVED** [review]
looks good

**Comment by aurobindoGupta** [comment]
resolve conflicts

**Pull request merged**